src/pages/en/guides/content-collections.md

```diff
@@ -0,0 +1,479 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Content Collections (Experimental)
+description: Content collections help organize your Markdown and type-check your frontmatter with schemas.
+i18nReady: false
+---
+
+Content collections help organize your Markdown or MDX and type-check your frontmatter with schemas. You may reach for collections if you:
+- **Plan to use content in multiple areas** of your site (landing pages, footers, navigation, etc).
+- **Have a non-trivial number of documents** to manage and fetch (example: a blog with 50+ posts).
+- **Want to enforce frontmatter fields,** and fail if fields are missing (example: every blog post should have a title and description).
+
+## Glossary
+
+We'll be using "schema," "collection," and "entry" throughout this guide. Let's define these terms:
+
+- **Schema:** the shape and type you expect your data to have. In this case, frontmatter data.
+- **Collection:** a set of data that share a common schema. In this case, a set of Markdown and MDX files.
+- **Entry:** A piece of data belonging to a given collection. In this case, a single Markdown or MDX file.
+
+## The content directory
+
+Content Collections introduce a new, reserved directory that Astro will manage for you: `src/content/`. This directory is where all content collections and frontmatter schemas live.
+
+
+## Creating a collection
+
+All entries in `src/content/` **must** be nested in a "collection" directory. This allows you to retrieve a collection of entries based on the directory name, and optionally enforce frontmatter types with a schema. In this way, collections are a bit like database tables or content types in a CMS: they help you group similar kinds of content together.
+
+What this looks like in practice:
+
+```bash "newsletter/" "blog/"
+src/content/
+  # All newsletters have the same frontmatter properties
+  newsletter/
+    week-1.md
+    week-2.md
+    week-3.md
+  # All blog posts have the same frontmatter properties
+  blog/
+    columbia.md
+    enterprise.md
+    endeavour.md
+```
```

Comment:
Circling back now I’ve realised this:

````suggestion
```bash "newsletter/" "blog/"
src/content/
│   # All blog posts have the same frontmatter properties
├── blog/
│   ├── columbia.md
│   ├── endeavour.md
│   └── enterprise.md
│   # All newsletters have the same frontmatter properties
└── newsletter/
    ├── week-1.md
    ├── week-2.md
    └── week-3.md
```
````